Mathlib/Data/Finsupp/Weight.lean

```diff
@@ -158,8 +158,9 @@ theorem weight_eq_zero_iff_eq_zero
     simp only [Finsupp.coe_zero, Pi.zero_apply]
     by_contra hs
     apply NonTorsionWeight.ne_zero w _
-    rw [← nonpos_iff_eq_zero, ← h]
-    exact le_weight_of_ne_zero' w hs
+    -- TODO: this is subtle, but I'm not sure what to best say in a comment...
```

Comment:
What's happening?

Reply:
Actually not much: I filled in the underscore and the second goal disappeared.